src/SingleSignatureDirtyHack.sol

```diff
@@ -269,4 +334,27 @@ abstract contract SingleSignatureDirtyHack is IERC5267, AbstractContext {
         transferParams.token.safeTransferFrom(transferParams.from, transferParams.to, transferParams.requestedAmount);
         return true;
     }
+
+    function transferFromPermit(
+        string calldata typeSuffix,
+        TransferParams calldata transferParams,
+        PackedSignature calldata sig
+    ) external preFlightChecklist(transferParams) returns (bool) {
```

Comment:
`consumeNonce` is inapplicable here, so we can't use `preFlightChecklist`

Reply:
right, didn't notice 🤦‍♂️